src/PageLayout/PageLayout.tsx

```diff
@@ -435,7 +445,6 @@ const Content: React.FC<React.PropsWithChildren<PageLayoutContentProps>> = ({
         {
           display: isHidden ? 'none' : 'flex',
           flexDirection: 'column',
-          order: REGION_ORDER.content,
```

Comment:
I think this is breaking storybooks. The "Custom Pane Widths" story seems to have the Pane position changed up.